apps/web/package.json

```diff
@@ -5,11 +5,11 @@
   "license": "MIT",
   "scripts": {
     "dev": "next dev --turbo",
-    "build": "next build",
+    "build": "next build && pnpm sitemap",
     "start": "next start",
     "lint": "next lint",
     "icons": "npx @svgr/cli --no-runtime-config --typescript --replace-attr-values \"#64de21=currentColor\" --out-dir icons/generated -- icons/svg && npx prettier icons/generated --write",
-    "postbuild": "next-sitemap",
+    "sitemap": "next-sitemap",
```

Comment:
We probably don't need this script and could update the `build` script to be `next build && next-sitemap`

Reply:
Ah gotcha. Let me try that!